ceph/ceph_admin/osd.py

```diff
@@ -78,6 +79,19 @@ def apply(self, config: Dict) -> None:
         if not config.get("args", {}).get("all-available-devices"):
             config["args"]["all-available-devices"] = True
 
+        # set ceph config for crimson-osd
```

Comment:
Can we add this as a separate test? I believe this would be temp settings. 

Adding here would make changes to the OSD deployment workflow. We can leave the OSD library as is. 

this workflow needed for crimson can be added as a separate test.

Reply:
This has been moved from osd library and included as individual steps during bootstrap process, these commands are not invoked from Test suite itself.